### ISSUE TITLE
Ignore commented lines for arrow alignment

### ISSUE THREAD

**Issue by ghost** [rodjek/puppet-lint#77]
When checking arrow alignment commented lines should be ignored. So the following example shouldn't give an error:

```
file { 'foo':
  ensure => directory,
  # purge => true,
}
```
